modules/api/go.mod

```diff
@@ -1,8 +1,8 @@
 module k8c.io/dashboard/v2
 
-go 1.22.8
+go 1.23.0
```

Comment:
Why do we need to bump the go version in release branch?

Reply:
This happened because of a Cilium update.

Reply:
Arguably by accident, but the cookie has crumbled and now it's too late. For the future we should/will be stricter about allowing Go minor upgrades in release branches, I hope.